packages/docusaurus-plugin-content-blog/src/blogUtils.ts

```diff
@@ -138,14 +195,17 @@ export async function generateBlogFeed(
   blogPosts.forEach((post) => {
     const {
       id,
-      metadata: {title: metadataTitle, permalink, date, description},
+      metadata: {title: metadataTitle, permalink, date, description, author},
     } = post;
     feed.addItem({
       title: metadataTitle,
       id,
       link: normalizeUrl([siteUrl, permalink]),
       date,
       description,
+      ...(author
+        ? {author: [{name: author.name, link: author.url}]}
+        : undefined),
```

Comment:
can feeds support multiple authors?

Reply:
At least the `feed` dependency does, but seems there's no standard way to handle this: https://stackoverflow.com/questions/5855993/multiple-authors-in-rss-or-atoms